Source/Fuse.Controls.Native/SystemScreenConfigBase.uno

```diff
@@ -0,0 +1,132 @@
+using Uno;
+using Uno.UX;
+using Uno.Collections;
+using Fuse;
+
+namespace Fuse.Controls
+{
+	/**
+		TODO write docs please
+	*/
+	public class SystemScreenConfigBase : Behavior
+	{
+		public enum Visibility
+	    {
+	        None,
+	        Minimal,
+	        Full,
+	    }
+
+	    private static List<SystemScreenConfigBase> rootedConfigs = new List<SystemScreenConfigBase>();
+
+	    protected Timer _timer;
+		private bool _rooted;
+
+	    protected override void OnRooted()
+		{
+			base.OnRooted();
+
+			_rooted = true;
+
+			rootedConfigs.Add(this);
+
+		}
+		protected override void OnUnrooted()
+		{
+			base.OnUnrooted();
+
+			resetTimer();
+
+			rootedConfigs.Remove(this);
+			_rooted = false;
+		}
+
+		protected double calculateResetTime() 
+		{
+			double lowestReset = _resetDelay;
+
+			foreach(SystemScreenConfigBase theBase in rootedConfigs) 
```

Comment:
Don't use `foreach`. Our memory manager doesn't deal with it well. Though it's not called much here I'd like to just keep it out of fuselibs due to the memory problem.

Use a classic C loop.